load-balanced-emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/read/LoadBalancedDynamoDBRecordReader.java

```diff
@@ -123,25 +131,11 @@ public void close() throws IOException {
     pageMux.setDraining(true);
   }
 
-  protected void convertDynamoDBItemToValue(Map<String, AttributeValue> item, V value) {
-    DynamoDBItemWritable ddbItem = new DynamoDBItemWritable(item);
-    convertDynamoDBItemToValue(ddbItem, value);
-  }
-
-  /**
-   * Convert a DynamoDB item to some value. Note that the nature of this method requires V to have
-   * some form of setter since the converted value must be stored in toValue.
-   *
-   * @param item    The item retrieved from DynamoDB
-   * @param toValue Where the converted item should be stored
-   */
-  protected abstract void convertDynamoDBItemToValue(DynamoDBItemWritable item, V toValue);
```

Comment:
what other classes implemented this?
Aren't we loosing support for other key types than Text ?

Reply:
The only implementation in emr-dynamodb-connector was `DefaultDynamoDBRecordReader`, which also fixes the key type to `Text`. Maybe there are other implementations in the wild…

Reply:
we need to focus just on what is in emr-dynamodb-connector